test/server/data/test-suites/call-fixture-from-async-function/testfile.js

```diff
@@ -0,0 +1,38 @@
+fixture('fixture 1');
```

Comment:
It would be nice to have tests for fixtures and tests with modifiers like `skip`, `before`, etc.